babel.config.js

```diff
@@ -1,7 +1,20 @@
 module.exports = function generateConfig(api) {
   api.cache(true);
   return {
-    presets: [['@babel/preset-env'], '@babel/preset-react', '@babel/preset-flow'],
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          targets: {
+            browsers: ['last 1 version', 'ie >= 11'],
```

Comment:
Isn't the default the last 2 versions? Do we want to lower the browser coverage here?

Reply:
cool, will fix